spec/features/scoping_methods_course_creation_spec.rb

```diff
@@ -0,0 +1,141 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+describe 'Course creation for Article Scoped Programs', type: :feature, js: true do
+  before do
+    TrainingModule.load_all
+    stub_oauth_edit
+
+    user = create(:user,
+                  id: 1,
+                  permissions: User::Permissions::INSTRUCTOR)
+    create(:training_modules_users, user_id: user.id,
+                                    training_module_id: 3,
+                                    completed_at: Time.zone.now)
+    allow(Features).to receive(:wiki_ed?).and_return(false)
+    login_as(user, scope: :user)
+
+    visit root_path
+    click_link 'Create Course'
```

Comment:
Where is this link? In P&E mode, the button on the home page should be called "Create an Independent Program" (which is also what the modal's header says once the course creator launches), right?

Reply:
fixed this